explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -102,14 +102,27 @@ object ExploreLayout:
               x.event match {
                 // TODO: Handle logout events
                 case ExploreEvent.PWAUpdateId =>
-                  toastRef
-                    .prompt(
-                      "A new version of explore is available!",
-                      IO(
-                        ctx.broadcastChannel.postMessage(ExploreEvent.PWAReload)
-                      ).runAsyncAndForget
-                    )
-                    .to[IO]
+                  // Clear other toasts first
+                  toastRef.clear().to[IO] *>
+                    toastRef
+                      .upgradePrompt(
+                        <.span(
+                          "A new version of ",
+                          <.a(
+                            ExploreStyles.UpgradeLink,
+                            "Explore",
+                            ^.href   := s"https://github.com/gemini-hlsw/explore/compare/${utils.gitHash.orEmpty}...HEAD",
```

Comment:
nice

Reply:
Let's experiment with this, it may not be the best for all users